packages/date/src/DateRange.js

```diff
@@ -252,6 +308,7 @@ const DateRange = ({
         }}
       >
         <DateRangePicker
+          renderMonthElement={renderMonthElement}
```

Comment:
`renderMonthElement` is placed above `{...datePickerProps}` just in case people are already passing in `datePickerProps.renderMonthElement` on their own.